### PR TITLE
check it

### PR DESCRIPTION
**Pull request by akachenia**

### CODE REVIEW

**Review by MustafaOk — COMMENTED** [review]
i dont know if i should approve it, I need more praising

**Comment by aliakpa** [comment]
sending more gas:)

**Review by MustafaOk — COMMENTED** [review]
gas me upp 👯

File: src/test/java/Mustafa_Aisal.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class Mustafa_Aisal {
+    public static void main(String[] args) {
+        System.out.println("Hi Mustafa, great working with you$");
+        System.out.println("Great Job Victor! He saved our life");
```
Comment (IgorTirsina) [inline src/test/java/Mustafa_Aisal.java:4]: Victor need to be a doctor, saved so many lives :)

**Review by IgorTirsina — COMMENTED** [review]
code is clean

**Pull request closed without merging**